### ISSUE TITLE
Figure out how to send RAD to test subjects

### ISSUE THREAD

**Issue by NunoAlexandre** [radicle-dev/radicle-registry#331]
Scope: next Tuesday's internal test run

**Comment by geigerzaehler** [comment]
One idea: Just put the hex encoded seed for `//Alice` here and we can add it to the JSON

**Comment by NunoAlexandre** [comment]
@geigerzaehler which JSON are you referring to?

**Comment by geigerzaehler** [comment]
> @geigerzaehler which JSON are you referring to?

The `accounts.json` file that is used by the CLI

**Comment by NunoAlexandre** [comment]
So you'd want the CLI to have the Alice super account in the accounts by default, in order to do what? It seems kind of weird to me to do that. Users will `cli account list` and have this account there. Or are you just suggesting we (@radicle-dev/registry) team to have that? I can only imagine the latter, so I will take care of that.

**Comment by CodeSandwich** [comment]
Another idea from yesterday: add a CLI command `radicle-registry-cli account transfer_from_dispenser`. It has all the options of `radicle-registry-cli account transfer` except it has no `tx_options` and `funds` capped on 1 million. Instead of using `author` from the `tx_options` it always uses the private key generated from `//Alice`.

**Comment by CodeSandwich** [comment]
Another idea from yesterday: add a CLI command `radicle-registry-cli account transfer_from_dispenser`. It has all the options of `radicle-registry-cli account transfer` except it has no `tx_options` and `funds` capped on 1 million. Instead of using `author` from the `tx_options` it always uses the private key generated from `//Alice`, just like in chain spec.

**Comment by NunoAlexandre** [comment]
Another idea: Transfer funds to an account when generated. That way, new accounts have funds right away and users don't need to follow any extra steps.

**Comment by CodeSandwich** [comment]
> Transfer funds to an account when generated.

Indeed, that would work. But it would change the logic and meaning of creation of an account depending on the chain. In the public network they must be created empty.

**Comment by NunoAlexandre** [comment]
That's true, but we are not managing multiple versions in parallel, that's why I find it acceptable and user-friendly.

**Comment by CodeSandwich** [comment]
Is this issue duplicate of https://github.com/radicle-dev/radicle-registry/issues/292?

**Comment by NunoAlexandre** [comment]
No, this one is focused on next week's internal test round. Discussed and solved aside.
